tailormap-components/projects/core/src/lib/shared/helpers/attribute-filter.helper.ts

```diff
@@ -63,6 +64,11 @@ export class AttributeFilterHelper {
     if (filter.condition === 'NOT_LIKE') {
       query.push('NOT');
     }
+    if (filter.condition === 'NULL') {
+      query.push('IS');
```

Comment:
query.push('IS NULL'); is the same and maybe a bit more readable

Reply:
Yes i thought the same but i was not sure if that's a problem because you have 2 indexes in the array instead of 3

Reply:
no, the array is just for building up a string and is joined with a space later on so does not matter how many items it contains